core/common/src/main/java/alluxio/underfs/BaseUnderFileSystem.java

```diff
@@ -62,7 +62,7 @@ public void configureProperties() throws IOException {
 
   @Override
   public OutputStream create(String path) throws IOException {
-    return create(path, CreateOptions.defaults());
+    return create(path, CreateOptions.defaults().setCreateParent(true));
```

Comment:
I agree it is a little weird. @ifcharming and I did discuss this. For permissions, it makes sense to have a default = false. For journal and other test cases, which also use the same UFS API and don't care about permissions, it makes sense for a default = true. The latter case ends up using create w/o options. The former uses create w/ options as permissions are set.